bin/genconfig2

```diff
@@ -5,7 +5,7 @@ $LOAD_PATH.unshift(lib_dir)
 
 require 'beaker-hostgenerator'
 
-STDERR.puts("Warning: 'genconfig2' is deprecated and will be removed in beaker-hostgenerator 1.x\n\n")
+$stderr.puts("Warning: 'genconfig2' is deprecated and will be removed in beaker-hostgenerator 1.x\n\n")
```

Comment:
If we're going to change this for the stderr stream, we should definitely change it for all the streams (so also stdout).

So we should change `STDOUT` to `$stdout` everywhere too.